docs/sources/explore/get-started-with-explore.md

```diff
@@ -0,0 +1,210 @@
+{{% /admonition %}}
+
+### Generate Explore URLs from external tools
+
+Because Explore URLs have a defined structure, you can build a URL from external tools and open it in Grafana. The URL structure is:
+
+```
+http://<grafana_url>/explore?panes=<panes>&schemaVersion=<schema_version>&orgId=<org_id>
+```
+
+where:
+
+- `org_id` is the organization ID
+- `schema_version` is the schema version (should be set to the latest version which is `1`)
+- `panes` is a URL-encoded JSON object of panes, where each key is the pane ID and each value is an object matching the following schema:
+
+```
+{
+  datasource: string; // the pane's root datasource UID, or `-- Mixed --` for mixed datasources
+  queries: {
+    refId: string; // an alphanumeric identifier for this query, must be unique within the pane, i.e. "A", "B", "C", etc.
+    datasource: {
+      uid: string; // the query's datasource UID ie: "AD7864H6422"
+      type: string; // the query's datasource type-id, i.e: "loki"
+    }
+    // ... any other datasource-specific query parameters
+  }[]; // array of queries for this pane
+  range: {
+    from: string; // the start time, in milliseconds since epoch
+    to: string; // the end time, in milliseconds since epoch
+  }
+}
+```
+
+{{< admonition type="note" >}}
+The `from` and `to` also accept relative ranges defined in [Time units and relative ranges](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/dashboards/use-dashboards/#time-units-and-relative-ranges).
+{{< /admonition >}}
+
+## Share shortened link
+
+{{< admonition type="note" >}}
+Available in Grafana 7.3 and later versions.
+{{< /admonition >}}
+
+The Share shortened link capability allows you to create smaller and simpler URLs of the format `/goto/:uid` instead of using longer URLs with query parameters. To create a shortened link to the executed query, click the **Share** option in the Explore toolbar.
+
+A shortened link that's not accessed automatically gets deleted after a [configurable period](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#short_links), which defaults to seven days. However, if the link is accessed at least once, it will not be deleted.
+
+### Share shortened links with absolute time
+
+{{< admonition type="note" >}}
+Available in Grafana 10.3 and later versions.
+{{< /admonition >}}
+
+<!-- Short links have two options: keeping relative time (for example, from two hours ago to now) or absolute time (for example, from 8am to 10am). Sharing a shortened link by default will copy the time range selected, relative or absolute. Clicking the dropdown button next to the share shortened link button and selecting one of the options under "Time-Sync URL Links" will allow you to create a short link with the absolute time - meaning anyone receiving the link will see the same data you are seeing, even if they open the link at another time. This will not affect your selected time range. -->
```

Comment:
Is this a html comment? will it render or what is the reason for it?

Reply:
Yeah I meant the html comment ( within the `<!-- --->` tags) just wasn't sure if that is something that will be rendered somehow or just forgotten.